hbase-shell/src/main/ruby/shell/commands/move_namespaces_rsgroup.rb

```diff
@@ -31,6 +31,8 @@ def help
 
       def command(dest, namespaces)
         rsgroup_admin.move_namespaces(dest, namespaces)
+        arg = {METHOD => 'set','hbase.rsgroup.name'=>dest}
+        admin.alter_namespace(namespaces[0], arg)
```

Comment:
Why we only alter the first one in array?

Reply:
BTW, is it possible to add ut for test?

Reply:
> Why we only alter the first one in array?

sorry,I missed something, pr updated already , please review again!